tests/support/activemq/queue_details.py

```diff
@@ -1,13 +1,20 @@
 from time import sleep
 
 import requests
+
+from django.conf import settings
 from parsel import Selector
+
 from tests.support.dtos import CurrentDestinationStatus
 
 
-def current_queue_configuration(queue_name, host="localhost") -> CurrentDestinationStatus:
+def current_queue_configuration(
+    queue_name: str,
+    host: str = settings.STOMP_SERVER_HOST,
+    port: str = settings.STOMP_SERVER_INTERFACE_PORT,
```

Comment:
**suggestion:** Same thing here. Since these params have a default value, the type could be `Optional`.